winfetch.ps1

```diff
@@ -572,14 +572,92 @@ function info_uptime {
 
 # ===== RESOLUTION =====
 function info_resolution {
-    Add-Type -AssemblyName System.Windows.Forms
-    $displays = foreach ($monitor in [System.Windows.Forms.Screen]::AllScreens) {
-        "$($monitor.Bounds.Size.Width)x$($monitor.Bounds.Size.Height)"
+    Add-Type -AssemblyName System.Core
+    Add-Type -AssemblyName System.Runtime.InteropServices
+    $id = Get-Random
+    $code = @"
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+public class Pinvoke$id
+{
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+    struct MONITORINFOEX
+    {
+        public int Size;
+        public Rect Monitor;
+        public Rect WorkArea;
+        public uint Flags;
+        [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 32)]
+        public string DeviceName;
+    }
+    
+    [DllImport("Shcore.dll")]
+    static public extern bool SetProcessDpiAwareness(int value);
+
+    [DllImport("user32.dll")]
+    static extern bool EnumDisplayMonitors(
+        IntPtr hdc,
+        IntPtr lprcClip,
+        EnumMonitorsDelegate lpfnEnum,
+        IntPtr dwData
+    );
+
+    [DllImport("user32.dll", CharSet = CharSet.Auto)]
+    static extern bool GetMonitorInfo(IntPtr hMonitor, ref MONITORINFOEX lpmi);
+
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Rect
+    {
+        public int left;
+        public int top;
+        public int right;
+        public int bottom;
+    }
+
+    delegate bool EnumMonitorsDelegate(IntPtr hMonitor, IntPtr hdcMonitor, ref Rect lprcMonitor, IntPtr dwData);
+
+    static private List<DisplayInfo> m_displays = new List<DisplayInfo>();
+    
+    static bool EnumProc(IntPtr hMonitor, IntPtr hdcMonitor, ref Rect lprcMonitor, IntPtr dwData)
+    {
+        MONITORINFOEX mONITORINFOEX = new MONITORINFOEX();
+        mONITORINFOEX.Size = Marshal.SizeOf(typeof(MONITORINFOEX));
+        GetMonitorInfo(hMonitor, ref mONITORINFOEX);
+
+        var displayInfo = new DisplayInfo();
+        displayInfo.X = (int)((mONITORINFOEX.Monitor.right - mONITORINFOEX.Monitor.left));
+        displayInfo.Y = (int)((mONITORINFOEX.Monitor.bottom - mONITORINFOEX.Monitor.top));
+
+        m_displays.Add(displayInfo);
+        return true;
+    }
+
+    public class DisplayInfo
+    {
+        public int X;
+        public int Y;
     }
 
+    public static string GetResult()
```

Comment:
Here too, a more descriptive name instead of `GetResult`.

We can also expose it so as to call it directly from outside instead of using Main.